packages/hasher/src/hasher.ts

```diff
@@ -1,8 +1,22 @@
-import { bufferFromString } from '@fuel-ts/crypto';
 import type { BytesLike } from '@fuel-ts/utils';
-import { arrayify, hexlify } from '@fuel-ts/utils';
+import { arrayify, concat, hexlify, toUtf8Bytes } from '@fuel-ts/utils';
 import { sha256 as sha256AsBytes } from '@noble/hashes/sha256';
 
+/**
+ * The prefix for the message to be hashed
+ */
+const MESSAGE_PREFIX = '\x19Fuel Signed Message:\n';
+
+/**
+ * When a string is provided, we hash as a UTF-8 string using SHA-256.
+ *
+ * When a `raw` object is provided, we hash using SHA-256 of the following format:
+ * ```console
+ * 0x19 <0x46 (F)> <uel Signed Message:\n" + len(message)> <message>
+ * ```
+ */
+export type HashableMessage = string | { raw: BytesLike };
```

Comment:
```suggestion
export type HashableMessage = string | { personal_sign: BytesLike };
```

Maybe `personal_sign` is more descriptive than raw?

<img width="1109" alt="image" src="https://github.com/user-attachments/assets/36acaea8-d945-43f2-9f33-d22036822805" />

Reply:
Best to use camel-cased `personalSign` to keep the standards, and for linting reasons.

It is essential to link the EIP docs in the TypeDocs block for this parameter.

Reply:
a7168b647ef9f3b05fd6d05041aa5cdfd0995dbe